setup.py

```diff
@@ -30,7 +30,10 @@
         'tabulate'
     ],
     url='https://github.com/dataxu/github-pr',
-    version='1.1.1',
+    version='2.0.0',
     scripts=['github_pr.py'],
     keywords=['github'],
+    dependency_links=[
+        "git+https://github.com/dataxu/PyGithub#egg=PyGithub-1.26.1"
```

Comment:
Agreed Corey, what would be the correct way to get our latest version of dataxu/PyGithub? I think the one thing that seperates us from the parent of our dataxu fork is the "squash to merge" which hasn't been merged into PyGithub yet.